content/en/dashboards/change_overlays/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Change Overlays
+kind: documentation
+description: Overlay your change events on graphs to correlate performance anomalies with changes in your application
+further_reading:
+- link: "/tracing/services/deployment_tracking/"
+  tag: "Documentation"
+  text: "Getting started with APM Deployment Tracking"
+- link: "https://www.datadoghq.com/blog/datadog-deployment-tracking/"
+  tag: "Blog"
+  text: "Monitor code deployments with Deployment Tracking in Datadog APM"
+- link: "https://www.datadoghq.com/blog/faulty-deployment-detection/"
+  tag: "Blog"
+  text: "Release code confidently with Automatic Faulty Deployment Detection"
+- link: "/real_user_monitoring/guide/setup-rum-deployment-tracking/?tab=npm"
+  tag: "Documentation"
+  text: "Getting started with RUM Deployment Tracking"
+- link: "https://www.datadoghq.com/blog/datadog-rum-deployment-tracking/"
+  tag: "Blog"
+  text: "Troubleshoot faulty frontend deployments with Deployment Tracking in RUM"
+
+
+---
+<div class="alert alert-warning">
```

Comment:
We have a more full-featured callout available that we usually use for beta features. You can see our internal docs on it here: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Callouts-(formerly-beta-callout), and an example here: https://docs-staging.datadoghq.com/aliciascott/DOCS-5770-community-integrations/account_management/plan_and_usage/cost_details/

Do you want to use the callout instead of the alert box?